### PR TITLE
feat(ImJoyPluginAPI): add setImagePiecewiseFunctionPoints and getter

### PR DESCRIPTION
**Pull request by PaulHax**

### CODE REVIEW

File: src/ImJoyPluginAPI.js [hunk 1]
```diff
@@ -253,6 +253,14 @@ class ImJoyPluginAPI {
     return this.viewer.getImagePiecewiseFunctionGaussians(component, name)
```
Comment (thewtex) [inline src/ImJoyPluginAPI.js:253]: @PaulHax These are no longer valid? They should be removed?

**Review by thewtex — COMMENTED** [review]
:

File: src/ImJoyPluginAPI.js (hunk 1 above)
Comment (PaulHax) [reply src/ImJoyPluginAPI.js]: Yes.  I just removed them.

**Review by thewtex — APPROVED** [review]
:rocket:

**Pull request merged**